.github/workflows/publish_latest.yml

```diff
@@ -0,0 +1,63 @@
+name: Publish Latest Cymetric
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/publish_latest.yml'
```

Comment:
Shouldn't we re-run on `Dockerfile` changes too?  Or would that be redundant to the fact that `build_test.yml` runs with `Dockerfile` changes?

Reply:
I just realized that none of these run if only the `Dockerfile` changes....  We probably need to run something

Reply:
> Shouldn't we re-run on `Dockerfile` changes too? Or would that be redundant to the fact that `build_test.yml` runs with `Dockerfile` changes?

It would be redundant, `build_test.yml` should run if we have a PR that changes the `Dockerfile` (even if that is the only file that changes, right?).  Then if a PR is merged new images will be published by `publish_latest.yml` on the push to `main`

Reply:
It doesn't look like `build_test` runs if there is ONLY a `Dockerfile` change - unusual but not impossible

Reply:
Scratch that! I was interpreting paths-ignore backwards